packages/contract-event-stream/__tests__/node_env.js

```diff
@@ -20,33 +19,22 @@ describe("contract-event-stream tests in node environment", () => {
   let artifact;
 
   beforeAll(async () => {
-    // 1. Compile contract artifact
-    const { SimpleStorageWithEvents } = await compile(
-      "SimpleStorageWithEvents.sol",
-    );
-
-    // 2. Spawn Ganache test blockchain
-    provider = Ganache.provider({ seed: "drizzle-utils" });
-    web3 = new Web3(provider);
-    accounts = await web3.eth.getAccounts();
-
-    // 3. Create initial contract instance
-    const instance = new web3.eth.Contract(SimpleStorageWithEvents.abi);
-
-    // 4. Deploy contract and get new deployed Instance
-    const deployedInstance = await instance
-      .deploy({ data: SimpleStorageWithEvents.bytecode })
-      .send({ from: accounts[0], gas: 150000 });
+    const { contractArtifact, ...rest } = await initTestChain({
```

Comment:
I don't like this pattern because the `...rest` is kind of distracting from its intention. I'd rather you make an intermediate variable here:

```
const testChain = await initTestChain()
contractArtifact = testChain.contractArtifact
```